pep257.py

```diff
@@ -100,7 +123,9 @@ class Module(Definition):
     _nest = staticmethod(lambda s: {'def': Function, 'class': Class}[s])
     module = property(lambda self: self)
     all = property(lambda self: self._all)
```

Comment:
Ha-ha, lambda assignments are not allowed, but property and staticmethod assignments are fine, amirite? :smile: